sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/enums/StaticIconName.java

```diff
@@ -0,0 +1,911 @@
+package com.smartdevicelink.proxy.rpc.enums;
+
+/**
+ * Enumeration of Static Icon Names
+ * @since SmartDeviceLink 4.5
```

Comment:
You can remove this @since. The static icons themselves have been around for a long time